### ISSUE TITLE
Parallelization Bug: multi_scan total_retry_count always returns nil

### ISSUE THREAD

**Issue by jcharr1** [lyndsey-ferguson/fastlane-plugin-test_center#124]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description

A small issue that still exists as of parallelization beta 9 is that `total_retry_count` is always nil at the end of a multi_scan run no matter what.

**Comment by lyndsey-ferguson** [comment]
Hm. I hadn't thought of how parallelization works with that. Would you just add up the retries across the batches?

**Comment by jcharr1** [comment]
That’s how I would do it.

**Comment by lyndsey-ferguson** [comment]
@jcharr1 can you test this fix?

Can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-124-total-retry-count-is-nill"
```
(yes, the branch name itself is misspelled)

Please attach the console output as a text file to this issue (makes it easier for me to review).

**Comment by lyndsey-ferguson** [comment]
[Here](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/issues/124) is the PR if you're curious

**Comment by jcharr1** [comment]
Here you go. At first glance it looks like it worked ok. BTW, `be` at the beginning is just an alias I made for `bundle exec` so I wouldn't have to type it out all the time.

[multi_scan-total_retries.txt](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/files/3473977/multi_scan-total_retries.txt)

**Comment by lyndsey-ferguson** [comment]
Wonderful, yes it did work. Thank you for testing!

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.8.0](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.8.0)
